Data/models/DataController.swift

```diff
@@ -162,3 +198,10 @@ public class DataController: NSObject {
         return backgroundContext
     }
 }
+
+fileprivate extension URL {
+    
+    func sqliteFiles(dbName: String) throws -> [URL] {
+        return try FileManager.default.contentsOfDirectory(at: self, includingPropertiesForKeys: nil, options: []).filter({$0.lastPathComponent.hasPrefix(dbName)})
```

Comment:
Won't that retrieve all files, not only with .sqlite extension?

Reply:
yes thats intended. I want to retrieve all files that have a name prefix `Brave.sqlite` like `Brave.sqlite-wal` etc. I think keeping this function in an extension may not be the best way, so moving it into the migration method itself.